### PR TITLE
refactor: user-consent 

### PR DESCRIPTION
**Pull request by Prerna290**
## Type
- ( ) Fix
- (X) Refactor
- ( ) Style
- ( ) Docs
- ( ) Feat
- ( ) Hotfix

## Screenshots
image
image
image
image
image
image

## Bundle Size
4.93 MB

### CODE REVIEW

File: apps/commudle-admin/src/app/feature-modules/public-community/public-community.module.ts [hunk 1]
```diff
@@ -33,6 +33,7 @@ import { PublicCommunityRoutingModule } from './public-community-routing.module'
 import { PublicCommunityNotificationsComponent } from './components/public-community-notifications/public-community-notifications.component';
 import { NotificationsModule } from 'apps/commudle-admin/src/app/feature-modules/notifications/notifications.module';
 import { FontAwesomeModule } from '@fortawesome/angular-fontawesome';
+import { UserConsentsComponent } from '../../app-shared-components/user-consents/user-consents.component';
```
Comment (Prerna290) [inline apps/commudle-admin/src/app/feature-modules/public-community/public-community.module.ts]: change path

File: apps/shared-modules/mini-user-profile/components/user-follow/user-follow.component.ts [hunk 2]
```diff
@@ -90,8 +91,19 @@ export class UserFollowComponent implements OnChanges, OnDestroy {
     );
   }
 
-  openDialog(ref: TemplateRef<any>) {
-    this.nbDialogService.open(ref);
+  // openDialog(ref: TemplateRef<any>) {
+  //   this.nbDialogService.open(ref);
+  //   this.gtm.dataLayerPushEvent('user-follow-initiate', { com_followee_id: this.user.id });
+  // }
```
Comment (ArshdeepGrover) [inline apps/shared-modules/mini-user-profile/components/user-follow/user-follow.component.ts:98]: why are we comment this?

File: apps/commudle-admin/src/app/feature-modules/public-community/components/membership-toggle/membership-toggle.component.ts [hunk 3]
```diff
@@ -29,6 +30,7 @@ export class MembershipToggleComponent implements OnInit {
   @Input() status: NbComponentStatus = 'basic';
   @Input() size: NbComponentSize = 'small';
   @Input() appearance: NbButtonAppearance = 'filled';
+  joinCommunity = false;
```
Comment (ArshdeepGrover) [inline apps/commudle-admin/src/app/feature-modules/public-community/components/membership-toggle/membership-toggle.component.ts:33]: let's write data type!

File: apps/commudle-admin/src/app/feature-modules/public-community/components/membership-toggle/membership-toggle.component.html [hunk 4]
```diff
@@ -1,5 +1,6 @@
+<!-- (click)="openDialog(addMemberDialog)" -->
```
Comment (ArshdeepGrover) [inline apps/commudle-admin/src/app/feature-modules/public-community/components/membership-toggle/membership-toggle.component.html:1]: why are we comment this?

File: apps/commudle-admin/src/app/feature-modules/email-confirmations/components/user-role-confirmation/user-role-confirmation.component.ts [hunk 5]
```diff
@@ -19,11 +21,13 @@ export class UserRoleConfirmationComponent implements OnInit, OnDestroy {
   event: IEvent;
   communityGroup: ICommunityGroup;
   EUserRoles = EUserRoles;
+  acceptRole = false;
```
Comment (ArshdeepGrover) [inline apps/commudle-admin/src/app/feature-modules/email-confirmations/components/user-role-confirmation/user-role-confirmation.component.ts:25]: Let's write data type!

**Review by ArshdeepGrover — CHANGES_REQUESTED** [review]

**Comment by ArshdeepGrover** [comment]
let's fill Bundle Size as per the PR template!

& also put mobile view screenshots

File: apps/commudle-admin/src/app/feature-modules/email-confirmations/components/user-role-confirmation/user-role-confirmation.component.ts (hunk 5 above)
Comment (Prerna290) [reply apps/commudle-admin/src/app/feature-modules/email-confirmations/components/user-role-confirmation/user-role-confirmation.component.ts:25]: Type boolean trivially inferred from a boolean literal, remove type annotation.

It shows this error

File: apps/commudle-admin/src/app/feature-modules/public-community/components/membership-toggle/membership-toggle.component.html (hunk 4 above)
Comment (Prerna290) [reply apps/commudle-admin/src/app/feature-modules/public-community/components/membership-toggle/membership-toggle.component.html:1]: I have not deleted any previous function, just commented it out, once the testing is done, i will remove the function through which the component was working earlier

File: apps/commudle-admin/src/app/feature-modules/public-community/components/membership-toggle/membership-toggle.component.ts (hunk 3 above)
Comment (Prerna290) [reply apps/commudle-admin/src/app/feature-modules/public-community/components/membership-toggle/membership-toggle.component.ts:33]: Type boolean trivially inferred from a boolean literal, remove type annotation.

It shows this error

Comment (Prerna290) [reply apps/commudle-admin/src/app/feature-modules/public-community/components/membership-toggle/membership-toggle.component.ts:33]: I have not deleted any previous function, just commented it out, once the testing is done, i will remove the function through which the component was working earlier

File: apps/shared-modules/mini-user-profile/components/user-follow/user-follow.component.ts (hunk 2 above)
Comment (Prerna290) [reply apps/shared-modules/mini-user-profile/components/user-follow/user-follow.component.ts:98]: I have not deleted any previous function, just commented it out, once the testing is done, i will remove the function through which the component was working earlier

**Pull request closed without merging**